src/client/websocket/WebSocketShard.js

```diff
@@ -435,7 +435,7 @@ class WebSocketShard extends EventEmitter {
       this.debug(`Tried to send packet ${data} but no WebSocket is available!`);
       return;
     }
-    this.ws.send(WebSocket.pack(data));
+    this.ws.send(WebSocket.pack(data), err => err && this.manager.client.emit(Events.ERROR, err));
```

Comment:
newlines are free and this is disgusting. use an if statement.

Reply:
this doesn't exist in browsers, since they just use the WebSocket error event. can you add a comment mentioning not to influence control flow here.